plugin/environment.go

```diff
@@ -94,6 +94,15 @@ func (env *Environment) IsActive(id string) bool {
 	return ok
 }
 
+// StaticFilesPath returns a path and true if the plugin with the given id is active.
+// It returns an empty string and false if the path is not set or invalid
+func (env *Environment) StaticFilesPath(id string) (string, bool) {
```

Comment:
nit: We're mixing the term "static" and "public" in this pull request. Propose we converge on public files everywhere to match the hard-coded path: e.g. `PublicFilesPath`.